frontend/components/Domain/Recipe/RecipePrintView.vue

```diff
@@ -216,12 +244,18 @@ export default defineComponent({
       return props.recipe.notes && props.recipe.notes.length > 0;
     });
 
+    const hasNutrition = computed(() => {
+      return props.recipe.nutrition && props.recipe.nutrition.length > 0;
+    });
+
```

Comment:
will always compute to false. Is also only used in the divider that is no needed.
```suggestion
```